src/main/java/com/github/manolo8/darkbot/config/tree/handlers/DropdownHandler.java

```diff
@@ -20,7 +20,7 @@ public static DropdownHandler of(Field field, PluginInfo namespace, PluginAPI ap
 
         Class<? extends Dropdown.Options<?>> optionCl = dropdown.options();
         LazyValue<Dropdown.Options<?>> options = optionCl != Dropdown.NullOptions.class ?
-                LazyValue.of(() -> api.requireInstance(optionCl)) :
+                LazyValue.of(() -> optionCl.isInterface() ? api.requireAPI(optionCl) : api.requireInstance(optionCl)) :
```

Comment:
interfaces should really *not* be supported for dropdowns, just have specific *instances* for the dropdowns, not apis

Reply:
Ok, I will remove it from PR. But I think that this suppliers can be implemented into managers too. I not understood why API interfaces cannot has dropdown option and you want keep implementation in API repo.
![image](https://github.com/darkbot-reloaded/DarkBot/assets/52008918/47753a86-aa66-4d97-93dd-152b9d3791b3)

Reply:
The shared package is still inside the API repo, not this one, but as an instance not as an interface, see https://github.com/darkbot-reloaded/DarkBotAPI/blob/master/shared/src/main/java/eu/darkbot/shared/config/NpcNames.java

Reply:
reverted